src/main/java/com/minecolonies/api/util/constant/IToolType.java

```diff
@@ -1,6 +1,9 @@
 package com.minecolonies.api.util.constant;
 
 import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+
+import java.util.Comparator;
 
 public interface IToolType
```

Comment:
If you're doing javadoc anyway, give the interface some love as well

Reply:
Done.